crates/compiler/fmt/src/expr.rs

```diff
@@ -654,6 +663,7 @@ fn fmt_apply(
 
     indent: u16,
     buf: &mut Buf<'_>,
+    use_parens: bool,
```

Comment:
This should be a local var (as discussed above). Also, slight preference to calling this `use_parens_and_commas` to more clearly distance this from the various `needs_parens` / `want_parens` bools floating around in the formatter.

Reply:
See above